src/_guides/language/sound-problems.md

```diff
@@ -55,9 +54,9 @@ see [Runtime errors](#common-errors-and-warnings).
 
 ### Undefined member
 
-<?code-excerpt "strong/analyzer-results-stable.txt" retain="/isn't defined for the type.*common_problems/" replace="/getter/<member\x3E/g; /'\w+'/'...'/g"?>
+<!-- code-excerpt "strong/analyzer-results-stable.txt" retain="/isn't defined for the type.*common_problems/" replace="/getter/<member\x3E/g; /'\w+'/'...'/g" -->
```

Comment:
Can we uncomment the code-excerpt? (I have the same question for every other commented-out code-excerpt in this file where we show the error text. :)